mythril/analysis/modules/integer.py

```diff
@@ -150,6 +148,29 @@ def _handle_sub(self, state):
         annotation = OverUnderflowAnnotation(state, "subtraction", c)
         op0.annotate(annotation)
 
+    def _handle_exp(self, state):
+        op0, op1 = self._get_args(state)
+        if op0.symbolic and op1.symbolic:
+            constraint = And(
+                op1 > symbol_factory.BitVecVal(256, 256),
+                op0 > symbol_factory.BitVecVal(1, 256),
+            )
+        elif op1.symbolic:
+            constraint = op1 >= symbol_factory.BitVecVal(
```

Comment:
There is an edge case where op0.value is either 0 or 1, which I think we would have problems with